src/nvim/macros.h

```diff
@@ -148,7 +148,9 @@
 /// zero in those cases (-Wdiv-by-zero in GCC).
 #define ARRAY_SIZE(arr) ((sizeof(arr)/sizeof((arr)[0])) / ((size_t)(!(sizeof(arr) % sizeof((arr)[0])))))
 
+#ifndef RGB
```

Comment:
our definition is not necessarily the same, and it is required.

Reply:
```
typedef unsigned long DWORD;
typedef DWORD COLORREF;
#define RGB(r,g,b) ((COLORREF)(((BYTE)(r)|((WORD)((BYTE)(g))<<8))|(((DWORD)(BYTE)(b))<<16)))
```
uint32_t equals unsigned long

Reply:
That's nice, but rather than depending on that, it could be undefined instead. That's what we do for other systems already.

Reply:
I got your means. thanks

Reply:
I mean we could `#undefine` it here.